101-data-factory-v2-blob-to-blob-copy/azuredeploy.json

```diff
@@ -1,209 +1,193 @@
+					"apiVersion": "2019-06-01",
+					"name": "[concat('default/', parameters('blobContainer'))]",
+					"dependsOn": [
+						"[parameters('storageAccountName')]"
+					]
+				}
+			]
+		},
+		{
+			"type": "Microsoft.DataFactory/factories",
+			"name": "[parameters('dataFactoryName')]",
+			"apiVersion": "2018-06-01",
+			"location": "[parameters('location')]",
+			"properties": {},
+			"identity": {
+				"type": "SystemAssigned"
+			},
+			"resources": [{
+					"type": "Microsoft.DataFactory/factories/linkedServices",
+					"name": "[concat(parameters('dataFactoryName'), '/ArmtemplateStorageLinkedService')]",
+					"apiVersion": "2018-06-01",
+            		"location": "[parameters('location')]",
+					"properties": {
+						"type": "AzureBlobStorage",
+						"typeProperties": {
+							"connectionString": "[concat('DefaultEndpointsProtocol=https;AccountName=',parameters('storageAccountName'),';AccountKey=',listKeys(variables('storageAccountId'), '2019-06-01').keys[0].value)]"
+						}
+					},
+					"dependsOn": [
+						"[parameters('dataFactoryName')]",
+						"[parameters('storageAccountName')]"
+					]
+				},
+				{
+					"type": "Microsoft.DataFactory/factories/datasets",
+					"name": "[concat(parameters('dataFactoryName'), '/ArmtemplateTestDatasetIn')]",
+					"apiVersion": "2018-06-01",
+            		"location": "[parameters('location')]",
+					"properties": {
+						"linkedServiceName": {
+							"referenceName": "ArmtemplateStorageLinkedService",
+							"type": "LinkedServiceReference"
+						},
+						"type": "Binary",
+						"typeProperties": {
+							"location": {
+								"type": "AzureBlobStorageLocation",
+								"container": "[parameters('blobContainer')]",
+								"folderPath": "input",
+								"fileName": "emp.txt"
+							}
+						}
+					},
+					"dependsOn": [
+						"[parameters('dataFactoryName')]",
+						"[variables('storageLinkedService')]"
+					]
+				},
+				{
+					"type": "Microsoft.DataFactory/factories/datasets",
+					"name": "[concat(parameters('dataFactoryName'), '/ArmtemplateTestDatasetOut')]",
+					"apiVersion": "2018-06-01",
+            		"location": "[parameters('location')]",
+					"properties": {
+						"linkedServiceName": {
+							"referenceName": "ArmtemplateStorageLinkedService",
+							"type": "LinkedServiceReference"
+						},
+						"type": "Binary",
+						"typeProperties": {
+							"location": {
+								"type": "AzureBlobStorageLocation",
+								"container": "[parameters('blobContainer')]",
+								"folderpath": "output"
+							}
+						}
+					},
+					"dependsOn": [
+						"[parameters('dataFactoryName')]",
+						"[variables('storageLinkedService')]"
+					]
+				},
+				{
+					"type": "Microsoft.DataFactory/factories/pipelines",
+					"name": "[concat(parameters('dataFactoryName'), '/ArmtemplateSampleCopyPipeline')]",
+					"apiVersion": "2018-06-01",
+            		"location": "[parameters('location')]",
+					"properties": {
+						"activities": [{
```

Comment:
format the JSON (e.g. SHIFT+ALT+F in VS Code)

Reply:
I did not see this off-formating when checking VS Code.

Reply:
line 147 shows a [ and a { on the same line (this is visible in VS Code).

Reply:
I was focused on location being out of place.

Corrected. :)